tests/unit/many-array-test.js

```diff
@@ -0,0 +1,233 @@
+import { moduleForModel, test } from 'ember-qunit';
+import compute from 'ember-macro-test-helpers/compute';
+import { getBy } from 'ember-awesome-macros';
+import array from 'ember-awesome-macros/array';
+import raw from 'ember-macro-helpers/raw';
+import { run } from '@ember/runloop';
+
+let parent, child1, child2;
+
+moduleForModel('parent', 'Unit | ManyArray', {
```

Comment:
Looks like the rest of the app now uses the new test syntax. Can you run the conversion here too please?

Reply:
yep, done :)